content/en/docs/ops/common-problems/observability-issues/index.md

```diff
@@ -27,6 +27,18 @@ The Mixer default installations include a Prometheus adapter and the configurati
 
 If the Istio Dashboard or the Prometheus queries don’t show the expected metrics, any step of the flow above may present an issue. The following sections provide instructions to troubleshoot each step.
 
+### Verify Istio CNI pods are running (if used)
+
+The Istio CNI plugin performs the Istio mesh pod traffic redirection in the Kubernetes pod lifecycle’s network setup phase, thereby removing the NET_ADMIN capability requirement for users deploying pods into the Istio mesh. The Istio CNI plugin replaces the functionality provided by the istio-init container.
```

Comment:
```suggestion
The Istio CNI plugin performs the Istio mesh pod traffic redirection in the Kubernetes pod lifecycle’s network setup phase, thereby removing the `NET_ADMIN` capability requirement for users deploying pods into the Istio mesh. The Istio CNI plugin replaces the functionality provided by the istio-init container.
```

Reply:
I've made another commit with these suggestions, as well as a link to the setup docs for the `NET_ADMIN` capability requirement.